pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -78,10 +79,10 @@ const (
 type CheckWrapper struct {
 	ID             string                       `yaml:"id" json:"id"`
 	Text           string                       `json:"d"`
-	Type           string                       `json:"-"`
+	Type           string                       `json:"tt"`
 	Remediation    string                       `json:"r"`
 	State          State                        `json:"s"`
-	Scored         bool                         `json:"-"`
+	Scored         bool                         `json:"sc"`
```

Comment:
Do we need to set this?

Reply:
hm the first time I tried it I had made this change, I can try without it

Reply:
nevermind, I meant where are we assigning a value to this - I see it is done in https://github.com/rancher/security-scan/blob/a1771f4ac81091e73723168014ae0cd93f04021d/pkg/kb-summarizer/summarizer/summarizer.go#L504